metacatalog/models/entry.py

```diff
@@ -615,7 +616,7 @@ def details_table(self, fmt='html'):
         else:
             raise ValueError("fmt has to be in ['html', 'latex', 'markdown']")
 
-    def add_details(self, details=None, commit=False, **kwargs):
+    def add_details(self, details=None, commit: bool = False, **kwargs):
```

Comment:
I made the same comment before, would'n we give a typehint for return `None` here?  
`-> None`